pom.xml

```diff
@@ -5,22 +5,25 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.49</version>
+    <version>4.29</version>
     <relativePath />
   </parent>
-  <groupId>org.jenkins-ci.plugins</groupId>
+
   <artifactId>webhook-step</artifactId>
   <version>1.5-SNAPSHOT</version>
```

Comment:
So https://github.com/jenkinsci/webhook-step-plugin/blob/30e3cb0c455e05bcadf5fe5f03ce0c74fdd3929b/.mvn/maven.config#L1-L3 and https://github.com/jenkinsci/webhook-step-plugin/blob/30e3cb0c455e05bcadf5fe5f03ce0c74fdd3929b/.mvn/extensions.xml#L4 are still unused.

Reply:
see https://www.jenkins.io/doc/developer/plugin-development/incrementals/ + https://www.jenkins.io/doc/developer/publishing/releasing-cd/

Reply:
Thank you to point me to this, @jglick. I learned something new. (although, still scratching my head and trying to be sure I understood it correctly)
Will study this (and the CD process) more in details.

TIL: don't copy configurations from another project without understanding enough of what they mean...

Reply:
And remember that without `cd.enabled = true` in RPU you will not be able to deploy.